### PR TITLE
Add ability to specify operator version to install

### PR DESCRIPTION
**Pull request by theemj**

### CODE REVIEW

**Comment by theemj** [comment]
Just for context: the reason the falcon and k10 operators can still be installed with automatic install plan approval is that they are both uncertified and used for negative test cases. falcon-operator is from the community-operators catalog and k10-kasten-operator hasn't been certified for any OCP version since 4.5, so whatever version of these we install will be fine.

File: tests/affiliatedcertification/tests/affiliated_certification_operator.go [hunk 1]
```diff
@@ -68,30 +74,40 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
...
 				affiliatedcertparameters.CertifiedOperatorGroup,
 				affiliatedcertparameters.OperatorSourceNamespace,
+				"dell-csi-operator-certified.v1.2.0",
+				v1alpha1.ApprovalManual,
 			)
 			Expect(err).ToNot(HaveOccurred(), "Error deploying operator "+
-				affiliatedcertparameters.CertifiedOperatorPrefixKubeturbo)
+				affiliatedcertparameters.CertifiedOperatorPrefixDellCSI)
+
+			time.Sleep(affiliatedcertparameters.DeploymentWait)
```
Comment (kononovn) [inline tests/affiliatedcertification/tests/affiliated_certification_operator.go:92]: Please remove sleep and add some Eventually loop instead

File: tests/affiliatedcertification/tests/affiliated_certification_operator.go [hunk 2]
```diff
@@ -105,9 +121,20 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 				affiliatedcertparameters.TestCertificationNameSpace,
 				affiliatedcertparameters.CertifiedOperatorGroup,
 				affiliatedcertparameters.OperatorSourceNamespace,
+				"artifactory-ha-operator.v1.1.20",
+				v1alpha1.ApprovalManual,
 			)
 			Expect(err).ToNot(HaveOccurred(), "Error deploying operator "+
 				affiliatedcertparameters.CertifiedOperatorPrefixArtifactoryHa)
+
+			time.Sleep(affiliatedcertparameters.DeploymentWait)
```
Comment (kononovn) [inline tests/affiliatedcertification/tests/affiliated_certification_operator.go:130]: Same here. Please get rid of time.Sleep function

File: tests/affiliatedcertification/affiliatedcerthelper/operator.go [hunk 3]
```diff
@@ -76,6 +79,56 @@ func DeployOperator(namespace string, subscription *v1alpha1.Subscription) error
...
+
+	if err != nil {
+		return fmt.Errorf("failed to update InstallPlan: %w", err)
+	}
+
+	return nil
+}
+
 // IsOperatorInstalled validates if the given operator is deployed on the given cluster.
 func IsOperatorInstalled(namespace string, csvPrefix string) error {
```
[the remark this answers is not in the document]
Comment (theemj) [reply tests/affiliatedcertification/affiliatedcerthelper/operator.go:127]: In the interest of getting 4.0.0 tested ASAP I'll address this along with the BeforeEach cleanup in later PRs, if that's ok.

**Pull request merged**